examples/input.rs

```diff
@@ -0,0 +1,316 @@
+//! A simple example that demonstrates capturing window and input events.
```

Comment:
Hey! This example is great! I like it!

There are just some minor details that I would change.